.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -1,5 +1,142 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  IMAGE_WIN_2016: "family/core-windows-2016"
+  IMAGE_WIN_2019: "family/core-windows-2019"
+  IMAGE_WIN_2022: "family/core-windows-2022"
+  IMAGE_MACOS_13: "generic-13-ventura-x64"
```

Comment:
Can we make it consistent like in https://github.com/elastic/beats/pull/37283/files#diff-18f9ca90b8c92c28fd6c97266f3b8214cbf81cdb37ed36784930965cdc8bf13eR9?